RobloxFanboy86#8963 + demoman boober#2239 (copy of PR #2565 but now its from upstream)

hammmurabi opened the pull request:
-Full content for Mandate of Palestine, including full Mandate collapse content for Palestine and Israel.

Reposting now that i'm a contributor and have access like you told me to.
Requesting review - DM me on discord for issues

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy86#8963

-Started focus tree for Mandatory Palestine
- Added starting idea and Arab Revolt ideas

Loc for most added stuff

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/RobloxFanboy86/1956_beta_robloxfanboy into Beta-Edits

hammmurabi committed:
RobloxFanboy#8963

Added balance of power for independence route (Loc and gfx needs to be done)

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy86#8963

-added some more focuses and loc
-added some charecters

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/RobloxFanboy86/1956_beta_robloxfanboy into Beta-Edits

hammmurabi committed:
Update MPL_traits.txt

fixed modifiers

hammmurabi committed:
RobloxFanboy#8963

-Added new focuses and ideas for independace path (Needs loc)

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy#8963

focus and gfx fixes

hammmurabi committed:
RobloxFanboy86#8963

-Added new focuses and ideas

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Update r56_mandate_of_palestine.txt

hammmurabi committed:
RobloxFanboy86#8963

Loc
Events for britain regarding independence

more stuff

hammmurabi committed:
RobloxFanboy#8936

-Cosmetic tag for free levant
-pretty much just that

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/RobloxFanboy86/1956_beta_robloxfanboy into Beta-Edits

hammmurabi committed:
RobloxFanboy#8963

New focuses for MPL

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy86#8963

-Mandate collapse
-Israel Focus modifications

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy#8963

Fixed some ideas and bop

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/RobloxFanboy86/1956_beta_robloxfanboy into Beta-Edits

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/RobloxFanboy86/1956_beta_robloxfanboy into Beta-Edits

hammmurabi committed:
RobloxFanboy86#8963

-Palestine tree
-Jordan oob fix
-loc changes

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
RobloxFanboy86#8963

Added focuses and corresponding ideas and events

hammmurabi committed:
RobloxFanboy#8963

added gfx and decisions for aliyah

hammmurabi committed:
RobloxFanboy86#8963

Aliyah Crisis decisions
loc
party names

hammmurabi committed:
RobloxFanboy#8963

focuses

hammmurabi committed:
Merge branch 'deliciousmods:Beta-Edits' into Beta-Edits

hammmurabi committed:
bruh

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
bruh 2

aqui tenenos

hammmurabi committed:
einstein mikunis

SkyHighGuydemomanboober committed:
Added mod files in the corresponding areas (followed naming convention)

I moved the files from my mod into the right folders, renamed the files of pre-existing PAL files, that had shared ISR and PAL stuff, including the loc of said file from PAL_l_english to PAL_ISR_l_english, and the other file named palestine into israel, considering it had the israeli ideas. I did that to make my files which are for palestine to be named as such. +added the portraits of the leaders/generals in gfx/interface/PAL/leaders.

ilkober committed:
Merge branch 'Beta-Edits' into pr/2520

hammmurabi committed:
i should have commited this 2w ago kill me

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
kms

hammmurabi committed:
sus

SkyHighGuydemomanboober committed:
Second try at mod integration, Here goes!

I have updated the files and put them in their respective places with the naming conventions, hopefully fixed the idea icons not working, added a general + portrait, fixed all the bugs that illya told me to fix, added decisions for coring stuff in the caliphate path since thats the most difficult thing for the path, occupying stuff, works like saudi arabia's coring decisions, added history/units and history/countries for palestine, refurbished event localisation, as a wise man once said "Localisation shall be in the Localisation folder"

hammmurabi committed:
h

SkyHighGuydemomanboober committed:
Fixed decisions and decision categories, added checks for all wargoals.

Just that, the tree will be ready after that, (atleast if illya doesnt take another look and there are 28 other errors for me to take care of!)

SkyHighGuydemomanboober committed:
removed an unnecessary original_tag visible trigger

thats literally all lol

SkyHighGuydemomanboober committed:
Update palestine.txt

SkyHighGuydemomanboober committed:
Merge branch 'PalestineTree' of https://github.com/SkyHighGuydemomanboober/1956_beta into PalestineTree

hammmurabi committed:
Merge remote-tracking branch 'upstream/Beta-Edits' into Beta-Edits

hammmurabi committed:
t

ilkober committed:
Merge branch 'Beta-Edits' into pr/2520

ilkober committed:
Merge branch 'PalestineTree' of https://github.com/SkyHighGuydemomanboober/1956_beta into pr/2520

hammmurabi committed:
MERGE WITH PALESTINE TREE

hammmurabi committed:
s

SkyHighGuydemomanboober committed:
fixed idea icons, patched up events

SkyHighGuydemomanboober committed:
fixed idea icons

SkyHighGuydemomanboober committed:
fixed ideas

SkyHighGuydemomanboober committed:
third try at integration, made 21 more decisions regarding divided military, and 3 categories housing them.

added 3 decisions for how to deal with the divided military in the state, one de-politicizing it, (more PP cost but slightly better stats), one islamicising it, and one putting the tribes in charge, each path puts the 3 dominant factions in charge, the nashashbi tribes, the holy war army, or the fidaiyya's emergence, (depoliticizing the army), aswell as fixed a problem for a focus that was supposed to give fighters

SkyHighGuydemomanboober committed:
missed a small lil underscore there

SkyHighGuydemomanboober committed:
got some New Ways portraits, credited in naming convention.

SkyHighGuydemomanboober committed:
Update PAL_l_english.yml

SkyHighGuydemomanboober committed:
fixed some stuff

SkyHighGuydemomanboober committed:
Merge branch 'PalestineTree' of https://github.com/SkyHighGuydemomanboober/1956_beta into PalestineTree

ilkober committed:
Merge branch 'Beta-Edits' into pr/2520

SkyHighGuydemomanboober committed:
bug-fixes PT.1

SkyHighGuydemomanboober committed:
bug-fixes PT.2

ilkober committed:
Merge branch 'Beta-Edits' into pr/2520

SkyHighGuydemomanboober committed:
Merge branch 'PalestineTree' of https://github.com/SkyHighGuydemomanboober/1956_beta into PalestineTree

SkyHighGuydemomanboober committed:
icon fixes

SkyHighGuydemomanboober committed:
bugfixes PT.3

bug fixes, checks for all wargoal focuses, revamped some of the focuses in the democratic path, also sanity checks for all factions, and finally, event naming convention fixed

SkyHighGuydemomanboober committed:
second loc fix

SkyHighGuydemomanboober committed:
bugfixes PT.4

SkyHighGuydemomanboober committed:
army tree rework

SkyHighGuydemomanboober committed:
bugfixes PT.5

SkyHighGuydemomanboober committed:
Military tree, and military junta path revamped, added high army chief and high command, also some advisors

SkyHighGuydemomanboober committed:
bugfixes PT.6

SkyHighGuydemomanboober committed:
event fixes + naval tree added to army tree

SkyHighGuydemomanboober committed:
finished revamping caliphate path, so it isnt just going to war with the world

SkyHighGuydemomanboober committed:
final revamp, revamped communist tree

SkyHighGuydemomanboober committed:
"needs a little focus spacing" - The Brit

ilkober committed:
Merge branch 'Beta-Edits' into pr/2520

SkyHighGuydemomanboober committed:
major bug fixing, fixed ALL sanity checks, also alternative for allies

SkyHighGuydemomanboober committed:
Merge branch 'PalestineTree' of https://github.com/SkyHighGuydemomanboober/1956_beta into PalestineTree

SkyHighGuydemomanboober committed:
COMPLETE INTEGRATION OF MANDATORY PALESTINE

MYEYESHURT

SkyHighGuydemomanboober committed:
more changes

SkyHighGuydemomanboober committed:
work

hammmurabi committed:
Merge branch 'PalestineTree' into Beta-Edits

hammmurabi committed:
Merge pull request #3 from deliciousmods/Beta-Edits

s

hammmurabi committed:
stuff

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Beta-Edits

SkyHighGuydemomanboober committed:
british puppet tree done (FINALLY FREE)

SkyHighGuydemomanboober committed:
better event system for palestine tree

SkyHighGuydemomanboober committed:
FINALLY FIXED SINAI BUG

hammmurabi committed:
so much shit

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Beta-Edits

SkyHighGuydemomanboober committed:
characters, yippee

SkyHighGuydemomanboober committed:
finished decisions for BOP

hammmurabi committed:
sleep

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Beta-Edits

SkyHighGuydemomanboober committed:
fix #1

hammmurabi committed:
new world events + focus rebalance

SkyHighGuydemomanboober committed:
Communist path, finished and playtested

SkyHighGuydemomanboober committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Beta-Edits

SkyHighGuydemomanboober committed:
final touches and last minute fixes

hammmurabi committed:
Final balancing and bug fixes

SkyHighGuydemomanboober committed:
fully removed cores for mpl, like illya said

hammmurabi committed:
focus rebalance

hammmurabi committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Palestine-Tree-Beta-Edits

hammmurabi committed:
Requests

SkyHighGuydemomanboober committed:
have fun robloxfanboy

hammmurabi committed:
kill me

SkyHighGuydemomanboober committed:
finished the missions for sinai war

SkyHighGuydemomanboober committed:
Merge branch 'Beta-Edits' of https://github.com/SkyHighGuydemomanboober/1956_beta into Beta-Edits

hammmurabi committed:
sanity checks + communist faction focus

hammmurabi committed:
bug fixes + tech inheritance fix + loc and gfx changes

hammmurabi committed:
character fixes + alan cunningham maybe?

hammmurabi committed:
Mandate Tree Expansion

hammmurabi committed:
moved traits + grenfell trait

marijn211 committed:
🤓error log

marijn211 commented on common/on_actions/r56_on_actions.txt:
At some point changes on the main to this file have been discarded, needs to be undone

hammmurabi replied:
resolved

marijn211 commented on common/decisions/PAL.txt:
I removed copy pasted formable decisions with minimal edits that caused error log, instead, use if-clauses in the availability checks of the original decision to make them different specificly for Palestine if that's all your changing

SkyHighGuydemomanboober committed:
final touches for the british path

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

TheBrit7 commented:
Some graphical file names such as portraits and focus icons are inconsistent and do not follow our standard naming process.

SkyHighGuydemomanboober committed:
bugfixes part 1

SkyHighGuydemomanboober committed:
fixes fixes fixes

SkyHighGuydemomanboober committed:
balastayen bug fixes

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
fixed on actions deleted stuff (why github?)

hammmurabi committed:
fix on actions *again*

hammmurabi committed:
Moved victory points to their actual locations

hammmurabi committed:
(not finished) map changes and border war fix

hammmurabi committed:
border war event fix + more map stuff

hammmurabi committed:
i love map modding i love map modding

hammmurabi committed:
update

update

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Update unitstacks.txt

hammmurabi committed:
death irl

SpicyAlfredo committed:
Some simple adjustments to start date sitution

hammmurabi committed:
Feedback from SpicyAlfredo

marijn211 committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Jordan and Mandate independence news events

SkyHighGuydemomanboober committed:
better military tree lots of polishing

SkyHighGuydemomanboober committed:
fixed all events for demanding stuff

SkyHighGuydemomanboober committed:
bruh

SkyHighGuydemomanboober committed:
removed/improved some filler focuses

hammmurabi committed:
polish changes and decisions fix

SkyHighGuydemomanboober committed:
fixed war support bug

hammmurabi committed:
balance and mikunis portrait

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
palestine focus tree tweaks

SkyHighGuydemomanboober committed:
polishing

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
more changes from feedback

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Focus search filters for MPL tree

marijn211 committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

marijn211 committed:
Excessive brackets removed

hammmurabi committed:
Update MPL_l_english.yml

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
small feedback from the brit

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
MORE polishing

SkyHighGuydemomanboober committed:
minor tweaks

hammmurabi committed:
initial independence preperation mechanic

hammmurabi committed:
Israel Flavour

SkyHighGuydemomanboober committed:
quality fixes, islamism ideology,  interwar pal flag, doesnt make sense how PAL is democracy in the war

SkyHighGuydemomanboober committed:
islamism.mp4

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
debug decisions, more flavour, loc, fixes

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Finished independence prep decisions and some other stuff

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Military tree expansion, random BoP events, mandate collapse rebalance

hammmurabi committed:
Connect order level to raid events

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Skeleton for foreign support mechanic and Israel war tree

hammmurabi committed:
first iteration of equipment purchase system (no ai yet)

SkyHighGuydemomanboober committed:
COMPLETE REVAMP OF THE PAL TREE

OH GOD NOT FAWZI AL-QAWUQJI

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
israel capital relocation

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
the removal of edginess + roblox tweaks

hammmurabi committed:
AI stuff for foreign support, israel capital move, i wanna die

hammmurabi committed:
chad (israel unfinished)

hammmurabi committed:
more icons and israel tree expansion hallo

SkyHighGuydemomanboober committed:
revamp

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
here you go roblox, good luck

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
Israel focus tree loc

SkyHighGuydemomanboober committed:
advisorinos

SkyHighGuydemomanboober committed:
set up skeleton for commie decisions so roblox just has to add events and simple var

SkyHighGuydemomanboober committed:
bruh

hammmurabi committed:
More israel stuff and fixes

SkyHighGuydemomanboober committed:
getoutofmyheadgetoutofmyheadgetoutofmyheadgetoutofmyhead

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
israel tree almost done + new gfx

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
israel focus tree finished

bruh

SkyHighGuydemomanboober committed:
poosh

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
omg bruh aww hell naw

hammmurabi committed:
decision fix

SkyHighGuydemomanboober committed:
bugfixes and commie tree revamp

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
more bugfixes + some characters

SkyHighGuydemomanboober committed:
br

hammmurabi committed:
Israel focus tree final tweaks and fixes

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
some fixes for PAL review

hammmurabi committed:
bruh

hammmurabi committed:
decision and loc fixes

SkyHighGuydemomanboober committed:
more more more bugfixes

SkyHighGuydemomanboober committed:
feedback from illya

hammmurabi committed:
MPL fixes + israel fixes

SkyHighGuydemomanboober committed:
illyawhenreview(loljktyt)

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
theorists and reformed jihad tree

hammmurabi committed:
fixed AI failing bop on historical

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
minor tweaks I HATE CHARACTER MODDING what

hammmurabi committed:
MPL character fixes

hammmurabi committed:
gfx

hammmurabi committed:
added focus to take back plaestine in case of white peacei  need to sleep help pls

hammmurabi committed:
israel tweaks after review by friend

hammmurabi committed:
more israel balance tweaks

hammmurabi committed:
Update MPL_l_english.yml

hammmurabi committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
naval tree, fixes and arab communism subideology

naval tree, (generic, just moved around focuses and loc) and made arab communism an ideology because arab socialism is just too un-radical stalinist and liberal for radwan al hilu

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

SkyHighGuydemomanboober committed:
sinai tweaks

SkyHighGuydemomanboober committed:
gg

hammmurabi committed:
bruh

hammmurabi committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

hammmurabi committed:
joe

hammmurabi committed:
fix israel communist flag

SkyHighGuydemomanboober committed:
hAEAEAGEAGAEGEA

SkyHighGuydemomanboober committed:
Merge branch 'mandate-palestine-tree-roblox-demoman' of https://github.com/deliciousmods/1956_beta into mandate-palestine-tree-roblox-demoman

marijn211 committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman

marijn211 committed:
Fixes

marijn211 committed:
More fix

marijn211 committed:
Merge branch 'Beta-Edits' into mandate-palestine-tree-roblox-demoman
